consul/tests/test_service.sls

```diff
@@ -0,0 +1,5 @@
+consul_service_running:
+  service.running:
+    - name: consul
+    - running: True
+    - enable: True
```

Comment:
This isn't a test. This is calling the service state from Salt to actually start and enable the specified service.